package.json

```diff
@@ -7,7 +7,7 @@
     "lint": "eslint src",
     "lint:fix": "eslint src --fix",
     "v:patch": "npm version --no-git-tag-version patch",
-    "build": "npm run v:patch && webpack -p",
+    "build": "webpack -p",
```

Comment:
without `npm version`, who will handle to create the tag and update the package.json version?

Reply:
If you want to publish a new release, you just have to run `npm version patch` or `npm run v:patch` and push the new commit + tags.

Having the version increment every time you try and build locally is kind of annoying imo.

Reply:
I mean, with the new workflow, if you need to publish the package on npm someone needs to update the version manually or it's handled by GH actions?

Reply:
Right now the steps are:

1. `npm run v:patch` to bump the version in package.json/package-lock.json and create a git tag
2. push the commit + new tag
3. go to github and draft a new release
4. select the new tag and add some release notes
5. hit publish                                                     `// after hitting this the "npm-publish.yml" GH action will publish to npm`

Does that clear things up? We could modify the workflow to skip steps 3-5 and let the GH action release as soon as the tags are pushed and add the release notes after.

Reply:
Ok, now it's clear. What about putting these steps in README? :)

Reply:
Sure I'll add a note somewhere